src/topology/algebra/ring.lean

```diff
@@ -77,6 +77,10 @@ instance prod_semiring {β : Type*}
   [semiring β] [topological_space β] [topological_semiring β] : topological_semiring (α × β) :=
 {}
 
+instance pi.topological_semiring {β : Type*} {C : β → Type*} [∀ b, topological_space (C b)]
+  [∀ b, ring (C b)] [∀ b, topological_semiring (C b)] : topological_semiring (Π b, C b) :=
```

Comment:
A small nit: `ring` and `topological_space` are data so using `Π` instead of `∀` is more reflective of how lean will pretty-print the lemma.
```suggestion
instance pi.topological_semiring {β : Type*} {C : β → Type*} [Π b, topological_space (C b)]
  [Π b, ring (C b)] [∀ b, topological_semiring (C b)] : topological_semiring (Π b, C b) :=
```

Reply:
Thanks, I changed it. I also replaced `ring` by `semiring` in `pi.topological_semiring`.